.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .vscode
+.env
```

Comment:
Can we move the .env file to the folder that needs it, and add the .gitignore there? Like if we only need the .env file in the backend-api, then move it there

Reply:
Turns out we can just have one `.env` file in the `blockchain` and use those env vars in `backend-api`! So this `.gitignore` file isn't needed, deleted.